docs/reference/contrib/forms/index.md

```diff
@@ -84,18 +92,20 @@ You now need to create two templates named ``form_page.html`` and ``form_page_la
             </form>
         </body>
     </html>
+```
 
-``form_page_landing.html`` is a standard Wagtail template, displayed after the user makes a successful form submission, `form_submission` will available in this template. If you want to dynamically override the landing page template, you can do so with the ``get_landing_page_template`` method (in the same way that you would with ``get_template``).
+`form_page_landing.html` is a standard Wagtail template, displayed after the user makes a successful form submission, `form_submission` will available in this template. If you want to dynamically override the landing page template, you can do so with the `get_landing_page_template` method (in the same way that you would with `get_template`).
```

Comment:
Nice that this fixes the typo on code formatting for `form_submission`!

```suggestion
`form_page_landing.html` is a standard Wagtail template, displayed after the user makes a successful form submission, `form_submission` will be available in this template. If you want to dynamically override the landing page template, you can do so with the `get_landing_page_template` method (in the same way that you would with `get_template`).
```

Reply:
Thanks for your help with this @thibaudcolas